src/t8_geometry/t8_geometry_implementations/t8_geometry_cad.cxx

```diff
@@ -40,6 +40,10 @@
 #include <TopoDS_Edge.hxx>
 #include <TopoDS_Face.hxx>
 #include <Standard_Version.hxx>
+#include <t8_schemes/t8_default/t8_default_prism/t8_dprism.h>
+
+const int t8_interpolation_coefficient_prism_edge[9] = { 1, 0, 0, 1, 0, 0, 2, 2, 2 };
```

Comment:
I don't think that these look-up-tables have to be global. They are only used in one function.